libc/src/__support/macros/config.h

```diff
@@ -28,6 +28,7 @@
 #define LIBC_HAS_FEATURE(f) 0
 #endif
 
+#ifndef __GNUC__
```

Comment:
`__GNUC__` is defined by many compilers, including Clang.
You need `#ifdef __clang__` here.

Reply:
Done.